### PR TITLE
Migrate Mozilla home page to Fluent (Fixes #9911)

### LINKED ISSUE #9911: Migrate legacy home page to Fluent
## Description

https://www.mozilla.org/es-ES/

I had put off filing this issue in the hope that this page would be redesigned, but it looks like that's not happening [at least in the short term](https://github.com/mozilla/bedrock/issues/9880). Holding off migrating these strings is now preventing removing the entire legacy .lang system for bedrock, and it's been three years since filing https://github.com/mozilla/bedrock/issues/5824.

Let's migrate these strings even though they are old, so we can move forward.

---

#### Success Criteria
- [ ] Page is live in production using Fluent along with existing migrations.

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
http://localhost:8000/es-ES/

## Issue / Bugzilla link
#9911

## Testing
```
./manage.py l10n_update
```

### CODE REVIEW

File: bedrock/mozorg/templates/mozorg/moss/base.html [hunk 1]
```diff
@@ -4,8 +4,6 @@
 
 {% extends "base-protocol-mozilla.html" %}
 
-{% add_lang_files "mozorg/moss/index" %}
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/moss/base.html:7]: Not related to this PR, but removing this line as no longer needed

File: bedrock/mozorg/templates/mozorg/home/home.html [hunk 2]
```diff
@@ -59,85 +51,66 @@
...
     image_url='img/home/dino.svg'
     )}}
 
     <section class="c-column-container">
       <div class="c-column">
         <div class="c-column-content">
-          <h2>{{ _('Our impact') }}</h2>
+          <h2>{{ ftl('home-our-impact') }}</h2>
           <p>
             {% if l10n_has_tag('home-update-20171116') %}
```
Comment (craigcook) [inline bedrock/mozorg/templates/mozorg/home/home.html:75]: Does `l10n_has_tag` still work with Fluent? I think this should be `ftl_has_messages`. 

BUT given how old these strings are maybe we don't need any conditional here and just go with the latest string and delete the other two.

**Review by craigcook — CHANGES_REQUESTED** [review]

File: bedrock/mozorg/templates/mozorg/home/home.html (hunk 2 above)
Comment (alexgibson) [reply bedrock/mozorg/templates/mozorg/home/home.html:75]: Good catch, will update 👍

**Comment by alexgibson** [comment]
@craigcook thanks for the review, removed the old conditional

**Review by craigcook — APPROVED** [review]

**Pull request merged**